ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
 
 - Security Standards
 - License Compliance Policies
-- Tooling
-  - Build in-house
-  - Adopt existing tools
-      - Commercial tools
-      - [Open Source tools](https://oss-compliance-tooling.org/Tooling-Landscape/OSS-Based-License-Compliance-Tools/)
-- Design OSS infrastructure
-
-### OS Community Education
 
-#### Create Internal OS Ambassadors
 
-- Drive event sponsorship
-- Train speakers for coding forums
-  
-#### Create educational programs
+### 📝 Establish and Improve Open Source Policies and Processess
+
+##### Policies
+
+- Using and compliance policy
+  - Forking an open source project
+  - Copying open source licensed files/snippets into organization's codebase
+  - Mixing code (owned and third-party OSS/proprietary) and distributing as products or services (sdk, REST API, SaaS, etc.)
+  - Using standalone open source tools (e.g. Firefox, MySQL, CentOS)
+- Policy on open source licenses
+- Policy on releasing code as open source
+  - Contributing back to third party OSS project
+  - Publishing a new open source project
+  - Contributing in personal time
+  - Signing contribution agreements (e.g. CLA or DCOs)
+- Policy on making open source business decisions
+  - Risk assessment and acceptance  
+  - Business alignment
+
+#### Processes
+
+- Creating Open Source
+  - Contributing to third-party OSS projects during work time
+    - Which copyright statement to use for contributions on behalf of the organization
+    - What to do if there is already a copyright notice in the code
+    - What to do if there is no copyright notice in the code
+    - What to do if the community object to any changes to the copyright notices
+  - Contributing to third-party OSS projects during personal time
+  - Publishing a new open source project
```

Comment:
I am sure that under this "Publishing a new open source project" there will be a lot of additional nodes. 

- Legal considerations
- IPR considerations
- Cleaning up the code from all internal references
- etc.

Reply:
@winterrocks  Expanded the section have a look.